AdminUi/apps/admin_ui/lib/components/shared/input_filter.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+
+class InputField extends StatefulWidget {
+  const InputField({required this.onEnteredText, required this.title, super.key});
+
+  final void Function(String enteredText) onEnteredText;
+  final String title;
+
+  @override
+  State<InputField> createState() => _InputFieldState();
+}
+
+class _InputFieldState extends State<InputField> {
+  final _searchAddressController = TextEditingController();
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      width: 180,
+      height: 70,
+      child: TextField(
+        controller: _searchAddressController,
```

Comment:
why do you need a controller here if you only have onChanged used and never the controller?

Reply:
I see your point, forgot to remove it becouse i tried using it before i refactored this input into a separate file.